coalib/bearlib/aspects/Spelling.py

```diff
@@ -22,6 +22,26 @@ class docs:
 
 
 @Spelling.subaspect
+class GeneralWordSpelling:
+    """
+    This aspect checks for misspelled words in your docs, given a specific
+    language.
+    """
+    class docs:
+        example = """
+        This is toatly wonrg.
+        """
+        example_language = 'English'
+        importance_reason = """
+        Words should always be written as they are supposed to be.
```

Comment:
generally all of those things seem rather trivial right? Esp since we're writing them. Problem is that the people who read this don't necessarily take tht for granted, otherwise we wouldn't need those fields.

It usually pays off to do a quick google search on every aspect and checking for some nice definitions and importance reasons, e.g. http://www.k5learning.com/blog/why-spelling-important